src/Facebook/Authentication/AccessToken.php

```diff
@@ -0,0 +1,223 @@
+<?php
+/**
+ * Copyright 2014 Facebook, Inc.
+ *
+ * You are hereby granted a non-exclusive, worldwide, royalty-free license to
+ * use, copy, modify, and distribute this software in source code or binary
+ * form for use in connection with the web services and APIs provided by
+ * Facebook.
+ *
+ * As with any software that integrates with the Facebook platform, your use
+ * of this software is subject to the Facebook Developer Principles and
+ * Policies [http://developers.facebook.com/policy/]. This copyright notice
+ * shall be included in all copies or substantial portions of the software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+namespace Facebook\Authentication;
+
+use Facebook\Exceptions\FacebookSDKException;
+
+/**
+ * Class AccessToken
+ * @package Facebook
+ */
+class AccessToken implements \Serializable
+{
+
+  /**
+   * The access token.
+   *
+   * @var string
+   */
+  protected $accessToken = '';
+
+  /**
+   * A unique ID to identify a client.
+   *
+   * @var string|null
+   */
+  protected $machineId;
```

Comment:
I think it's important to keep in since you can use it in the scenario were you have your app running on multiple clients (like web, native mobile, etc). And if the same access token is being used across multiple devices, it's in danger of getting flagged for spam. In that case you would store a `code` which you use to generate new long-lived access tokens and create a unique access token for each client you have. And that code-for-token feature uses the `machine_id` for added security. So I think we should keep this feature. :)